### ISSUE TITLE
Tooltips not visible on scroll

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/zerif-lite#494]
You should work on the zerif-lite-2 branch
I am unable to the read the whole info text for accessibility support as it disapears when I scroll.
